bin/jekyll_pages_api_search

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env ruby
+# Author: Mike Bland <michael.bland@gsa.gov>
+# Date:   2015-06-21
+
+# Monkeypatch to avoid "undefine method `render` error.
```

Comment:
So in 18F/jekyll_pages_api#27 I've updated `lib/jekyll_pages_api.rb` to explicitly include all of its components, and the problem appears to go away when I modify my Gemfile to use that updated version.